### PR TITLE
Fix checkout to existent PR

### PR DESCRIPTION
**Pull request by OlegZharkov**
This PR fixes checking out existent PR during content import.

here's an example https://github.com/OlegZharkov/content/pull/209/commits

### CODE REVIEW

File: scripts/git-utils/checkout-pr-if-exist.sh [hunk 1]
```diff
@@ -1,14 +1,19 @@
 #!/bin/bash
-# get all opened PRs  with specified label
 set -e
+# get all opened PRs with specified label
 RESULT=`curl -s https://api.github.com/search/issues?q=is:pr%20state:open%20label:$1%20repo:bio-tools/content`
 # check if PR is created by github-actions bot and return pull request-number
 PR_NUMBER=`jq '.items[]|select(.user.login == "github-actions[bot]")|.number' <<< $RESULT`
+# get the branch name and remove quotes
+TARGET_BRANCHNAME=`jq '.head.ref' <<< $(curl -s https://api.github.com/repos/OlegZharkov/content/pulls/$PR_NUMBER) | sed -r 's/^"|"$//g'`
```
Comment (hmenager) [inline scripts/git-utils/checkout-pr-if-exist.sh:8]: Is "OlegZharkov/content/" the right repo @OlegZharkov ?

**Review by hmenager — COMMENTED** [review]
Question about a weird repo queried through the API (see code comments)

File: scripts/git-utils/checkout-pr-if-exist.sh (hunk 1 above)
Comment (OlegZharkov) [reply scripts/git-utils/checkout-pr-if-exist.sh:8]: oh yeah, that was for debugging. Just a sec...

Comment (OlegZharkov) [reply scripts/git-utils/checkout-pr-if-exist.sh]: should be better now @hmenager

**Review by hmenager — APPROVED** [review]

**Pull request merged**